packages/wallet-adapter-react/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @aptos-labs/wallet-adapter-react
 
+## 3.1.6
+
+### Minor Changes
+
+- cb8acd1: Added the following headless React component and some utility functions to simplify the creation of new wallet adapter UIs:
```

Comment:
have you added it manually? we shouldn't. 

the way we handle changelog and releases is with changeset  https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/CONTRIBUTING.md#changesets

Reply:
Oops, I'll actually read the contributing doc and do it the right way

Reply:
The changesets have been added